tests/pyccel/test_pyccel.py

```diff
@@ -839,11 +839,12 @@ def test_function_aliasing():
             language = 'fortran')
 
 #------------------------------------------------------------------------------
+@pytest.mark.xdist_incompatible
 def test_inline(language):
     pyccel_test("scripts/decorators_inline.py", language = language)
 
 #------------------------------------------------------------------------------
-@pytest.mark.xfail(reason="Imported inline functions cannot import objects required for their contents")
+@pytest.mark.xdist_incompatible
 def test_inline_import(language):
     pyccel_test("scripts/runtest_decorators_inline.py",
             dependencies = ("scripts/decorators_inline.py"),
```

Comment:
Why are these functions incompatible with `xdist`?

Reply:
Both functions translate the file `scripts/decorators_inline.py` there is therefore a chance of them trying to modify the same file at the same time. Or (as I saw before I remembered to add it), the python result collection for `test_inline_import` may try to use the pyccelized version of `scripts.decorators_inline`. This is not what is intended and indeed will not work until issue #741 is fixed